Documentation/HowToGuides/AddingAnInteractor/README.md

```diff
@@ -0,0 +1,68 @@
+
+Drag and drop the `Interactions.Interactor` prefab to be a child of the `LeftControllerAlias` then drag and drop another `Interactions.Interactor` prefab to be a child of the `RightControllerAlias`.
+
+![Making Interactor Prefab AChild Of Left And Right Alias](assets/images/MakingInteractorPrefabAChildOfLeftAndRightAlias.png)
+
+Note: The following steps will be carried out on both the `LeftControllerAlias` and the `RightControllerAlias` but the instructions will only outline the steps once, so be sure to carry them out for both controller aliases.
+
+### Step 2
+
+Select the `Interactions.Interactor` prefab in the Unity Hierarchy under the `LeftControllerAlias` GameObject and change the `Interactor Facade` component to configure the base functionality of the Interactor.
+
+We need to specify an action that initiates the Interactor’s Grab notification. The Grab Action parameter on the Interactor Facade allows us to provide a Boolean Action to initiate the Grab function of the Interactor.
+
+For this example we will set up the pointer to use the simulator controls utilizing the mouse.
+
+Expand `CameraRigs.SpatialSimulator -> Input -> Control Objects Input -> Right Controller Input -> Buttons`.
+
+`Button One` is the left mouse button and we'll use this to activate the grab action on our left interactor so drag and drop the `Button One` GameObject into the `Grab Action` parameter on the Interactor Facade component on the `Interactions.Interactor` prefab nested inside `LeftControllerAlias` Gameobject
+
+![Drag And Drop Button One Into Grab Activation Parameter](assets/images/DragAndDropButtonOneIntoGrabActivationParameter.png)
+
+### Step 3
+
+When an Interactor grabs an interactable object, it is usually required that the velocity that the Interactor is moving at can be applied to the interactable object. For example, if we were to pick up an interactable object and throw it, then we would want the velocity that we’re moving our virtual controller at to be applied to the interactable object so we can control the power of the throw.
+
+The `Velocity Tracker` parameter on the `Interactor Facade` component allows us to specify a velocity tracking component that will provide the relevant velocity data. Fortunately, our TrackedAlias prefab already comes with the relevant `Velocity Tracker` components on the relevant aliases that can move freely in the virtual world such as the Headset, Left Controller and Right Controller.
+
+Drag and drop the `CameraRigs.TrackedAlias -> Aliases -> LeftControllerAlias` GameObject into the `Velocity Tracker` parameter on the Interactor Facade component for the `Interactions.Interactor` prefab nested under the `LeftControllerAlias` GameObject to set the interactor to track velocities from our Left Controller.
+
+![Drag And Drop Left Controller Alias Into Velocity Tracker Parameter](assets/images/DragAndDropLeftControllerAliasIntoVelocityTrackerParameter.png)
```

Comment:
Graphic is confusing again showing Interactor prefab as top level object instead of working on the ones within the tracked alias (graphic should show the left controller interactor being worked on).